pkg/clustermesh/common/remote_cluster.go

```diff
@@ -158,7 +158,12 @@ func (rc *remoteCluster) restartRemoteConnection() {
 					if backend != nil {
 						backend.Close()
 					}
-					rc.logger.WithError(err).Warning("Unable to establish etcd connection to remote cluster")
```

Comment:
@giorio94 I see you did not backport this change to v1.16. Was that intentional? I'm seeing it on downgrades to v1.16: https://github.com/cilium/cilium/actions/runs/11935307422/job/33266363458. Should I just allowlist it until v1.17 is out?

Reply:
Thanks for noticing. It seems I mistakenly linked a different backport to this one, and marked this one as completed (rather than the other one). Marking again for backport, so that it gets included in the next round.

Reply:
Aah, that's why the commits looked so different :laughing: